### PR TITLE
Fix(build):  Fixes broken build and also use GitHub Actions in place of travis

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit@chaosnative.com>
**What this PR does / why we need it**:

- This PR fixes the broken build due to the outdated node version used.
- Use GitHub Actions in place of Travis ci.
- The issue is the build was failing but the travis ci was passing. The failure was due to the older node version so updated it to version 14.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for [DCO](https://github.com/litmuschaos/litmus-docs/blob/master/CONTRIBUTING.md#sign-your-work) check to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Comment by uditgaurav** [comment]
- We need to set up the docker credentials as ENV `DNAME` and `DPASS`
- We have to replace the Travis with GitHub Actions as a mandatory check as the Travis is removed after this.

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -17,6 +17,11 @@ jobs:
           stable: 'false'
           go-version: '1.14.0'
 
+      - name: Links Verification
+        run: |
+          go get -u -v github.com/bmuschko/link-verifier
+          link-verifier --dirs website,README.md --fail=false
```
Comment (Jonsy13) [inline .github/workflows/pull_request.yml]: You may add the docs directory here as well. @uditgaurav

Comment (uditgaurav) [reply .github/workflows/pull_request.yml]: Added Thanks!

**Review by Jonsy13 — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -17,6 +17,11 @@ jobs:
           stable: 'false'
           go-version: '1.14.0'
 
+      - name: Links Verification
+        run: |
+          go get -u -v github.com/bmuschko/link-verifier
+          link-verifier --dirs website,README.md,docs --fail=false
```
Comment (ksatchit) [inline .github/workflows/pull_request.yml:23]: what about versioned docs? (we update links there retrospectively, i.e., after versioned doc creation - so there is scope for broken links)

Comment (uditgaurav) [reply .github/workflows/pull_request.yml:23]: versioned_docs will come under the website directory only. I have added website.

Comment (ksatchit) [reply .github/workflows/pull_request.yml:23]: versioned docs are inside website - missed that. Thanks!

**Review by k8s-dev — APPROVED** [review]
These changes looks good to me. Thanks.

**Pull request merged**